h2o-core/src/test/java/water/rapids/ast/prims/reducers/AstNaRowCntTest.java

```diff
@@ -0,0 +1,55 @@
+package water.rapids.ast.prims.reducers;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+import water.TestUtil;
+import water.fvec.Frame;
+import water.rapids.Rapids;
+import water.rapids.Val;
+import water.util.ArrayUtils;
+
+import static org.junit.Assert.assertEquals;
+
+
+/**
+ * Test the AstNaCnt.java class
+ */
+public class AstNaRowCntTest extends TestUtil {
```

Comment:
Please add some more tests with vecs of all types: categorical, string, time. We need to make sure the rapids function can understand NAs in all kinds of vecs.